autoBugTracker.py

```diff
@@ -0,0 +1,34 @@
+import argparse
+import executeUserScript
+
+
+class autoBugTracker(object):
```

Comment:
I believe we agreed on a more conventional name such as Main

Reply:
That was stretch goal when creating a package. I would discuss it again with product owner.